.github/workflows/pack.yml

```diff
@@ -0,0 +1,45 @@
+name: Pack extensions
+
+on:
+  push:
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [21.x]
+    container: archlinux:base-devel
```

Comment:
Why are you running the job inside of an Arch container on an Ubuntu runner? Wouldn't it be simpler to just build the extension on Ubuntu?